docs/Troubleshooting/Report-a-bug.md

```diff
@@ -3,11 +3,15 @@
 Bugs are errors in the system. If you find a bug in your Omeka Classic installation, please let us know.
 
 ## Before Reporting a Bug
-Always check to see if it has already been reported. You can search the [forum](http://forum.omeka.org) and our [github repositories](https://github.com/omeka) to see if the bug you have found has been identified.
+Always check to see if it has already been reported. You can search the [Forum](http://forum.omeka.org){target=_blank} and our [Github repositories](https://github.com/omeka){target=_blank} to see if the bug you have found has been identified. Feel free to respond to an existing issue to let us know that you are experiencing the same thing. 
 
 ## Who Posts Bugs and Where
 Users of varying technical skill will report bugs in different places. 
 
-We encourage novice users to post bugs to the [forums](http://forum.omeka.org) if they think somethings broken, but don't know why.
+We encourage novice users to post bugs to the [Omeka Classic Forum](https://forum.omeka.org/c/omeka-classic/7){target=_blank} if they think something is broken, but don't know why. If you believe it is an issue with a certain [plugin](https://forum.omeka.org/c/omeka-classic/plugins/10){target=_blank} or [theme](https://forum.omeka.org/c/omeka-classic/themes/11){target=_blank}, use the Forum categories for those. 
 
-Users who have an interest in hacking and developing code with Omeka, Power users and developers can report bugs directly as issues on [GitHub](https://github.com/omeka). Sometimes these bugs affect the core of the software, and sometimes they require basic modifications to a user's theme or plugin. Please use the correct repository for the bug you are reporting.
+Developers and other users who have an interest in programming can report bugs directly as issues on [GitHub](https://github.com/omeka){target=_blank}. Issues can be with the core software, or with a plugin or theme. Some plugins and themes are not maintained by the Omeka team, so use the [plugins directory](https://omeka.org/classic/plugins/){target=_blank} and [themes directory](https://omeka.org/classic/themes/){target=_blank} to find the correct Github repository to report the issue. 
+
+If you are unsure whether it is code within a plugin/theme or an interaction best solved with the core code itself, bring it to our attention in the [core Github repository for Omeka Classic](https://github.com/omeka/Omeka){target=_blank} first and we can direct you. Be careful not to post Classic issues in the S forum or repository, and vice versa. 
```

Comment:
I'd like @zerocrates to comment on whether we would like to steer these unsure users to the forums for first report rather than to the Omeka Classic github repo.

Reply:
Probably better that way.